rpc/dial_test.go

```diff
@@ -1150,10 +1150,11 @@ func TestDialMulticastDNS(t *testing.T) {
 
 	t.Run("fix mdns instance name", func(t *testing.T) {
 		rpcServer, err := NewServer(
-			logger,
+			logger.Named("server"),
```

Comment:
Not 100% intentional. I think* these are fine given the logger is golog and this is in tests and not called from RDK.

Reply:
> Any reason you needed to create better logger names here, though? Easier to read test output or something?

"easier to read" is a polite way of putting it. At least in the `authenticated` subtest, we make 3 separate dials. It was nigh impossible to parse which "base channel/stream" was client and which was server.